src/components/calendar/DateCalendar.tsx

```diff
@@ -88,13 +88,11 @@ const DateCalendar = (dateInfo: DateInfo) => {
   return (
     <>
       <Header title="Calendar" />
-      <MotionFadeDown css={commonStyle}>
-        <div>{`${year}년 ${month}월 ${day}일`}</div>
-        <div>
-          {`${dayOfWeek}${korean ? '요일 ' : 'DAY '} ${hour}시 ${min}분`}
-        </div>
-      </MotionFadeDown>
-      <MotionFadeDown css={calendarStyle}>
+      <MotionFade css={commonStyle}>
+        <div>{`${year}년 ${month}월 ${day}일 | ${dayOfWeek}${korean ? '요일 ' : 'DAY '} ${hour}시 ${min}분`}</div>
+        <div>{``}</div>
```

Comment:
```suggestion
const dayText = korean ? '요일 ' : 'DAY ';
        <div>{`${year}년 ${month}월 ${day}일 | ${dayOfWeek}${dayText} ${hour}시 ${min}분`}</div>
        <div>{``}</div>
```

[K4] : 렌더링 하는 곳에서 최대한 로직을 간단히 하기 위해 변수로 빼서 처리하는 것도 좋을 것 같습니다!